### PR TITLE
Patch/fix

### PR DESCRIPTION
**Pull request by aarti-joshi-nttd**

### CODE REVIEW

**Review by chris11-taylor-nttd — APPROVED** [review]

File: tests/post_deploy_functional/main_test.go [hunk 1]
```diff
@@ -31,11 +31,6 @@ func TestFirewallModule(t *testing.T) {
 		SetTestConfig(&testimpl.ThisTFModuleConfig{}).
 		SetTestConfigFolderName(testConfigsExamplesFolderDefault).
 		SetTestConfigFileName(infraTFVarFileNameDefault).
-		SetTestSpecificFlags(map[string]types.TestFlags{
-			"complete": {
```
Comment (debasish-sahoo-nttd) [inline tests/post_deploy_functional/main_test.go:35]: complete should be replaced with the actual example name "firewall_policy" in this case

File: tests/post_deploy_functional_readonly/main_test.go [hunk 2]
```diff
@@ -31,11 +31,6 @@ func TestFirewallModule(t *testing.T) {
 		SetTestConfig(&testimpl.ThisTFModuleConfig{}).
 		SetTestConfigFolderName(testConfigsExamplesFolderDefault).
 		SetTestConfigFileName(infraTFVarFileNameDefault).
-		SetTestSpecificFlags(map[string]types.TestFlags{
-			"complete": {
```
Comment (debasish-sahoo-nttd) [inline tests/post_deploy_functional_readonly/main_test.go:35]: complete should be replaced with the actual example name "firewall_policy" in this case

**Review by debasish-sahoo-nttd — CHANGES_REQUESTED** [review]

File: tests/post_deploy_functional/main_test.go (hunk 1 above)
Comment (aarti-joshi-nttd) [reply tests/post_deploy_functional/main_test.go:35]: these lines are deleted. so no need of fix. Thats a good catch. I have done that mistake at couple of other places, which are fixed now.

File: tests/post_deploy_functional_readonly/main_test.go (hunk 2 above)
Comment (aarti-joshi-nttd) [reply tests/post_deploy_functional_readonly/main_test.go:35]: same as above

**Review by debasish-sahoo-nttd — APPROVED** [review]

**Pull request merged**